images/dind/node/openshift-dind-lib.sh

```diff
@@ -12,17 +12,15 @@
 #  - 1: message indicating what conditions is being waited for (e.g. 'config to be written')
 #  - 2: a string representing an eval'able condition.  When eval'd it should not output
 #       anything to stdout or stderr.
-#  - 3: optional timeout in seconds.  If not provided, defaults to 60s.  If OS_WAIT_FOREVER
-#       is provided, wait forever.
+#  - 3: optional timeout in seconds.  If not provided, waits forever.
 # Returns:
 #  1 if the condition is not met before the timeout
-readonly OS_WAIT_FOREVER=-1
 function os::util::wait-for-condition() {
```

Comment:
I understand it's not new -- we need a better consensus about whether or not the DIND libs will be using code from `hack/lib` -- a conversation we need to have, but one we can have another day.